CREATE_YOUR_FIRST_EXTENSION.md

```diff
@@ -106,7 +106,7 @@ But it doesn't _do_ anything yet.  Here is where you can start defining your own
 If you want to handle REST Requests, implement the `ActionExtension` interface and override the `getExtensionRestHandlers()` method.  Pass a list of classes which will handle those requests.
 
 ```java
-import org.opensearch.sdk.ActionExtension;
+import org.opensearch.sdk.api.*;
```

Comment:
We prohibit star imports in our code, so just make this `org.opensearch.sdk.api.ActionExtension`.